data/keys.keymap

```diff
@@ -0,0 +1,45 @@
+﻿0:37
```

Comment:
It's better to add some comments here explaining each line like in TextIDs
e.g.:
1:39 //Function